samples/src/main/python/SynapseSample.py

```diff
@@ -16,6 +16,7 @@
             .format("com.microsoft.kusto.spark.synapse.datasource") \
             .option("spark.synapse.linkedService", "<link service name>") \
             .option("kustoDatabase", "<Database name>") \
+           #.option("authType", "LS") \ # if this added it uses the link service credentials, default is native - that will take the logged-in user credentials or the Synapse Workspace MSI for automated excution.            
```

Comment:
```suggestion
            #.option("authType", "LS") \ # If this is added, it uses the link service credentials. Otherwise, the default is native - using the logged-in user's credentials or the Synapse Workspace managed identity for automated execution.
```